contact-me.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Enquery Form </title>
+	<meta charset="utf-8">
+	<meta name="Description" content="This page show form to submit queries">
+	<meta name="keywords" content="nazish, contact">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+		<form>
+			<label for="name">Name:</label>
+			<input id="name" type="text" name="name" required><br>

```

Comment:
the `<br />` aren't necessary. 

I know you want to solve the visual ugliness with markup ... but refrain from doing so right now. CSS will make it pretty!